gradle-modules/src/main/kotlin/org.sonarsource.cloud-native.artifactory-configuration.gradle.kts

```diff
@@ -22,10 +22,6 @@ plugins {
     id("com.jfrog.artifactory")
 }
 
-if (project.path != ":") {
-    throw GradleException("This build script must be applied to the root project only")
-}
-
```

Comment:
I like this check because it prevent from mistakes. Maybe something like this will be good:
```
if (project.path != ":" || (project.path.startsWith(:private) && project.path.contains("-plugin"))) {
    throw GradleException("This build script must be applied to the root project or in \"-plugin\" module")
}
```